pom.xml

```diff
@@ -185,6 +185,9 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>0.8.5</version>
+                <configuration>
+                    <skip>${skipTests}</skip>
+                </configuration>
```

Comment:
This is what fixed the Heroku deployment. Heroku build with `-DskipTests=true` and without this we were getting some errors when JaCoCo was trying to write the report to the filesystem.